services/app-web/src/components/DoubleColumnRow/DoubleColumnRow.tsx

```diff
@@ -1,19 +0,0 @@
-import { Grid } from '@trussworks/react-uswds'
```

Comment:
Even though these components have changed a ton, I think it would be nice to have the git rename here instead of a delete and re-add